### DISCUSSION TITLE
brainflow with processing 4.3 (java)

### DISCUSSION THREAD

**Discussion by ankur-gupta-29** [brainflow-dev/brainflow#677] [General]
i want to use brainflow in processing 4.3 (java) .how can i use that

**Comment by Andrey1994** [comment]
I would say its a question for processing how the integrate 3rd party libraries...

It should be smth like this https://github.com/OpenBCI/OpenBCI_GUI/wiki/Developer-Setup and maybe @retiutut can guide you thorough this process if he has time and if you ask him politely)

This guide is specific for openbci gui and for processing 3 but overall process is similar with some changes in java versions, libraries locations, etc
